packages/textlint-formatter/package.json

```diff
@@ -16,7 +16,7 @@
     "bin/"
   ],
   "scripts": {
-    "test": "mocha test/*.js"
+    "test": "mocha test/**/*.js"
```

Comment:
Can you try --no-color option?

https://github.com/textlint/textlint/issues/402#issuecomment-352808734

Reply:
Run test from top directory without `--no-color` have passed successfully.

Reply:
Should I add `--no-color` option to the `package.json`, or leave it as it is (at this moment, and fix test cases later)?

Reply:
Please add `--no-color` option.
master branch should be always passed tests in any environment.